src/commands/tokens.py

```diff
@@ -106,6 +106,21 @@ def print_tokens_pretty(tokens):
       click.style(SEPARATOR, fg=COLOR_SEPARATOR)
     )
 
+
 def print_tokens_csv(tokens):
-  # TODO: Implement here the CSV formatting
-  click.echo("Not implemented yet")
+
+  writer = get_csv_writer()
+
+  writer.writerow(['ID', 'Symbol', 'Name', 'Decimals', 'Registered', 'Transaction'])
+
+  for token in tokens:
+    symbol = token.get('symbol', '')
+    name = token.get('name', '')
+
+    writer.writerow([token['id'],
+      symbol,
+      name,
+      token['decimals'],
+      token['address'],
+      format_date_time(token['create_date']),
```

Comment:
Maybe for CSV makes sense to use a ISO format? I would need to check how they behave importing in Numbers/Excel/GoogleDrive it would be great if they understand the date type and allow to do filtering and sorting